bond/package.xml

```diff
@@ -1,31 +1,34 @@
 <?xml version="1.0"?>
-<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>bond</name>
-  <version>1.8.3</version>
+  <version>1.8.4</version>
   <description>
     A bond allows two processes, A and B, to know when the other has
     terminated, either cleanly or by crashing.  The bond remains
     connected until it is either broken explicitly or until a
     heartbeat times out.
   </description>
   <maintainer email="michael@openrobotics.org">Michael Carroll</maintainer>
-  <license>BSD</license>
-
+  <license>Apache License 2.0</license>
   <url type="website">http://www.ros.org/wiki/bond</url>
   <url type="bugtracker">https://github.com/ros/bond_core/issues</url>
   <url type="repository">https://github.com/ros/bond_core</url>
   <author>Stuart Glaser</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
-
-  <build_depend>message_generation</build_depend>
-  <build_depend>std_msgs</build_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>std_msgs</buildtool_depend>
+  <buildtool_depend>builtin_interfaces</buildtool_depend>
```

Comment:
`std_msgs` and `builtin_interfaces` should be `build` and `exec` depends.

Reply:
And not `buildtool_depend`

Reply:
@mjcarroll I have made changes as per suggestion